spec/integrations/base_spec.rb

```diff
@@ -186,6 +186,22 @@
     end
   end
 
+  describe 'date_format option' do
+    let(:blueprint) do
+      Class.new(Blueprinter::Base) do
+        field :date, datetime_format: '%FT%T%:z'
+      end
+    end
+
+    context 'field with nil instead of value' do
+      let(:object) { OpenStruct.new(date: nil) }
+
+      it 'renders null' do
+        expect(blueprint.render_as_hash(object)).to eq(date: nil)
+      end
+    end
+  end
```

Comment:
There's already a context for datetime. Would you augment that context to include this test instead of putting this test here? 

https://github.com/procore/blueprinter/blame/master/spec/integrations/shared/base_render_examples.rb#L53-L64

Reply:
Done)